languagetool-language-modules/ga/src/main/resources/org/languagetool/resource/ga/disambiguation.xml

```diff
@@ -0,0 +1,379 @@
+            </marker>
+        </pattern>
+        <disambig action="remove" postag="(?:C[UMC]:)?Verb:.*:Imper.*"/>
+    </rule>
+    <!--
+    <rule id="NI_FHEICIM" name="ní fheicim">
+        <pattern>
+            <marker>
+                <token postag="Part:Vb:Neg">ní</token>
+                <token postag="(?:C[UMC]:)?Verb:.*:Neg.*" postag_regexp="yes"><exception regexp="yes">mh?aith</exception><exception scope="next" postag="Prep:Rel|Pron:Prep:.*" postag_regexp="yes">le</exception></token>
+            </marker>
+        </pattern>
+        <disambig action="filterall"></disambig>
+    </rule>
+    -->
+    <rule id="COS_AR_BOLG" name="cos ar bolg">
+        <pattern>
+            <marker>
+                <token postag="Noun:Fem:.*" postag_regexp="yes">cos</token>
+                <token postag="Prep:Simp">ar</token>
+                <token postag="Noun:Masc:Com:Sg">bolg</token>
+            </marker>
+        </pattern>
+        <disambig action="filterall"></disambig>
+    </rule>
+    <rule id="OS_ARD" name="os ard">
+        <pattern>
+            <marker>
+                <token postag="Prep:Simp">os</token>
+                <token postag="Noun:Masc:Com:Sg">ard</token>
+            </marker>
+        </pattern>
+        <disambig action="filterall"></disambig>
+    </rule>
+    <rulegroup id="NI_BA_COMP" name="ní ba mhó">
+        <rule>
+            <pattern>
+                <marker>
+                    <token postag="Subst:Noun:Sg:Part:Comp">ní</token>
+                    <token postag="Part:Sup">ba</token>
+                    <token postag="Adj:Comp:Len"></token>
+                </marker>
+            </pattern>
+            <disambig action="filterall"></disambig>
+        </rule>
+        <rule>
+            <pattern>
+                <marker>
+                    <token postag="Subst:Noun:Sg:Part:Comp">ní</token>
+                    <token>b</token>
+                    <token>'</token>
+                    <token postag="Adj:Comp:Len"></token>
+                </marker>
+            </pattern>
+            <disambig action="filterall"></disambig>
+        </rule>
+        <rule>
+            <pattern>
+                <marker>
+                    <token postag="Part:Comp" regexp="yes">ní b[a']</token>
+                    <token postag="Adj:Comp:Len"></token>
+                </marker>
+            </pattern>
+            <disambig action="filterall"></disambig>
+        </rule>
+    </rulegroup>
+    <!--
+    <rule id="GO_RAIBH" name="go raibh">
+        <pattern>
+            <marker>
+                <token postag="Prep:Vb:.*" postag_regexp="yes">go</token>
+                <token postag="Verb:.*" postag_regexp="yes">raibh</token>
+            </marker>
+        </pattern>
+        <disambig action="filterall"></disambig>
+    </rule>
+    -->
+    <rule id="NUM_DIG_ORD" name="6ú">
+        <pattern>
+            <marker>
+                <token regexp="yes">([0-9]+)ú</token>
```

Comment:
(nitpick) useless capture with parenthesis. Regexp engines can often be slower when capturing.